group-coordinator/src/main/java/org/apache/kafka/coordinator/group/classic/ClassicGroup.java

```diff
@@ -1376,6 +1378,7 @@ public Map<String, byte[]> groupAssignment() {
     public static ClassicGroup fromConsumerGroup(
```

Comment:
nit: I just found out that we have no unit tests for this method in ClassicGroupTest. It would be great to add some. What do you think?

Reply:
Yep somehow I missed adding unit tests for it. Let me add them for `fromConsumerGroup` and `fromClassicGroup` as well